src/ifElse.test.js

```diff
@@ -1,11 +1,45 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  let firstCallback;
+  let secondCallback;
 
+  beforeEach(() => {
+    firstCallback = jest.fn();
+    secondCallback = jest.fn();
   });
 
   // write tests here
+  it(`should be declared`, () => {
+    expect(ifElse)
+      .toBeInstanceOf(Function);
+  });
+
+  it('should return undefined', () => {
```

Comment:
The test description 'should return undefined' is not very descriptive. A better description might be 'should return undefined when no callbacks are provided' or 'should return undefined when the condition is true but the first callback does not return a value'. Try to make your test descriptions as clear as possible.